code/game/antagonist/station/inquisitor.dm

```diff
@@ -79,7 +79,7 @@
 	else
 		to_chat(player, "You have been working undercover here, until a signal from NT command calls you to action. You may wish to make your presence known to the local preacher, if there is one.")
 
-	to_chat(player, "You will need a ritual book to utilise your abilities. They can be found or purchased in the chapel. The Bounty ritual can be used to request items from central command. You may request a Priest upgrade to promote a new preacher.")
```

Comment:
Pretty sure "central command" here refers to inquisitor high command, rather than New Rome

Reply:
I think that distinction will be lost on most. The change is fine

Reply:
> Pretty sure "central command" here refers to inquisitor high command, rather than New Rome

Sure, you may argue that it's some inquisition ruling body, that apparently also a logistic hub for some reason, but in SS13 "Central Command" is associated with NanoTrasen's NAS Crescent, and I'd rather not have that association here.